linera-indexer/lib/src/common.rs

```diff
@@ -51,10 +51,10 @@ pub enum IndexerError {
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +50]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->```gt_suggestion
    CloneWithRootKey,
```<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/2915?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->